exp/operator/controllers/consts.go

```diff
@@ -21,5 +21,5 @@ import "time"
 const (
 	// preflightFailedRequeueAfter is how long to wait before trying to reconcile
 	// if some preflight check has failed.
-	preflightFailedRequeueAfter = 30 * time.Second
+	preflightFailedRequeueAfter = 5 * time.Second
```

Comment:
this was probably because of tests, i'll follow up and see what happens again if i revert this..
if it's tests, i'll make this a "var" and change it in suite_test.go

Reply:
this seems to not be an issue any more, i'll revert.